osu.Game/Overlays/Volume/VolumeMeter.cs

```diff
@@ -395,6 +398,8 @@ private void updateSelectedState()
                     selectedGlowContainer.FadeOut(transition_length, Easing.OutExpo);
                     break;
             }
+
+            hoverSample?.Play();
```

Comment:
Probably should be in the `Selected` case only, no? Else this will play twice when switching from one to another.